### PR TITLE
Subgraph: Several fixes

### PR DESCRIPTION
**Pull request by fabriziovigevani**
- Updates organization.active logic
- Fixes error when tokens address null

### CODE REVIEW

File: packages/subgraph/src/processors.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Address, BigInt, DataSourceTemplate } from '@graphprotocol/graph-ts'
+import { Address, BigInt, Bytes, DataSourceTemplate } from '@graphprotocol/graph-ts'
```
Comment (0xGabi) [inline packages/subgraph/src/processors.ts:1]:
```suggestion
import { Address, BigInt, DataSourceTemplate } from '@graphprotocol/graph-ts'

File: packages/subgraph/manifest/templates/contracts/DisputableVoting.template.yaml [hunk 2]
```diff
@@ -18,7 +18,9 @@
       - name: ConvictionVoting
         file: ./abis/ConvictionVoting.json
       - name: Agreement
-        file: ./abis/Agreement.json
+        file: ./abis/Agreement.json     
+      - name: ERC20
+        file: ./abis/ERC20.json
       - name: MiniMeToken
         file: ./abis/MiniMeToken.json
```
Comment (0xGabi) [inline packages/subgraph/manifest/templates/contracts/DisputableVoting.template.yaml:25]:
```suggestion
      - name: MiniMeToken
         file: ./abis/MiniMeToken.json

File: packages/subgraph/src/processors.ts [hunk 3]
```diff
@@ -26,7 +26,8 @@ export function processApp(orgAddress: Address, appAddress: Address, appId: stri
 
 export function processOrg(orgAddress: Address, timestamp: BigInt): void {
   const org = loadOrCreateOrg(orgAddress)
-  org.createdAt = timestamp
-  org.active = true
-  org.save()
+  if (org.createdAt.isZero()) {
```
Comment (0xGabi) [inline packages/subgraph/src/processors.ts:29]: Nice!

**Review by 0xGabi — APPROVED** [review]
LGTM!

File: packages/subgraph/manifest/templates/contracts/DisputableVoting.template.yaml (hunk 2 above)
Comment (0xGabi) [reply packages/subgraph/manifest/templates/contracts/DisputableVoting.template.yaml:25]: I think we don't need the ERC20 contract for the disputable voting.

Comment (fabriziovigevani) [reply packages/subgraph/manifest/templates/contracts/DisputableVoting.template.yaml:25]: Ah no, in this is case needed as we use the ERC20 abi when loading the token at `loadTokenData`

**Pull request merged**